x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/Section.tsx

```diff
@@ -15,7 +15,7 @@ interface Props {
 }
 
 export function Section({ keyValuePairs }: Props) {
-  if (keyValuePairs) {
+  if (keyValuePairs && keyValuePairs.length > 0) {
```

Comment:
nit: a slightly more readable approach is using `isEmpty`:

```js
import { isEmpty } from 'lodash';
if (!isEmpty(keyValuePairs)) {
```

Reply:
<img width="1200" alt="Screenshot 2019-11-13 at 09 41 01" src="https://user-images.githubusercontent.com/55978943/68746853-bb901900-05f9-11ea-87b4-640356c8d52d.png">

@sqren I noticed that this error happens when I use lodash.isEmpty. Do you know how to solve it?

Reply:
Oh, that's disappointing that `isEmpty` doesn't have better type guards.
Anyway, I think the type for `keyValuePairs` should not be an optional:

![image](https://user-images.githubusercontent.com/209966/68751037-30b31c80-0601-11ea-9e9d-fa29ce3d15d2.png)

So you should be able to change it to:
```ts
interface Props {
  keyValuePairs: KeyValuePair[];
}
```

Reply:
Which in turn will let you do:
```ts
  if (keyValuePairs.length > 0) {
    return <KeyValueTable keyValuePairs={keyValuePairs} />;
  }
```

or:
```
  if (!isEmpty(keyValuePairs)) {
    return <KeyValueTable keyValuePairs={keyValuePairs} />;
  }
```

Whatever you prefer.